### DISCUSSION TITLE
How to get tables by tag

### DISCUSSION THREAD

**Discussion by Ronaldo-spec** [open-metadata/OpenMetadata#7736] [General]
I try to get and showing tables that related with same Tag in Openmetadata with Laravel+PHP using API. But im so confused, keep getting error/failed. Do Openmetadata team have any documentaion or solution about that?

**Comment by Sachin-chaurasiya** [comment]
Hello @Ronaldo-spec , Welcome to OpenMetadata Community.
Can you share more details like How you are trying to get the tables by tag and what errors you are getting?

**Comment by Ronaldo-spec** [comment]
We have a website project for coursework based on Laravel-PHP that including OpenMetadata features. The plan is, we can replicate OpenMetadata features on the website.

Example : 
1. We succeed to make a page that showing databases/tables using API

I try to using documentation by swagger as reference

The Error :

1. I make tag category called "db" and tag name "pembayaran", so the table has tag "db.pembayaran".

   a. Try to check table with q=* the value is 20
![db pembayaran20](https://user-images.githubusercontent.com/55744190/194799655-2bf4ff3e-f301-4108-8f18-55d38ee57ff7.png)

   b. When i try to get table by tag, the value is 0
![db pembayaran](https://user-images.githubusercontent.com/55744190/194799176-db3c20e2-3112-489e-a774-8afd3dff73b7.png)

2. My pipeline counted by OpenMetadata, but when we open it, nothing inside
![pipeline1](https://user-images.githubusercontent.com/55744190/194800201-bc40258e-6e56-49c6-9c86-af9222f720ff.png)
![pipeline2](https://user-images.githubusercontent.com/55744190/194800207-3b907241-600d-4861-b735-7416ee15096b.png)
    Should we use Python instead of PHP to make it detected? Or do OpenMetadata have an example source codes? Because, once i try to install OpenMetadata from git clone, It can run but we cant see the source code if im not mistaken/forgot
    
    *Sorry for my bad english
    Thank you

**Reply by Ronaldo-spec** [reply]
Forget about this one :
My Airflow project inside are about ETL from database 1 to database 2 with PHP

**Reply by ayush-shah** [reply]
Hey @Ronaldo-spec 
can you try reindexing Elasticsearch?
this would work if you are on 0.12.1
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/40225091/194803643-e131a567-8e1b-4c2e-b675-5d393385f85d.png">

**Reply by Ronaldo-spec** [reply]
@ayush-shah Thank You, Sir. Maybe i will learn about it first.

Should it 0.12.1 version? look like ours is 0.12.0
if so, it really make us amazed little bit because we should reinstall it in the Docker to update the version, i guess. Remake all the databases, schema, data, and etc :"D

Oh, basically we are college student, this project for our report as an end semester project :D. really sorry from us if our explanation seems abstract/unclear, because OpenMetadata included as new service at our college

**Reply by ayush-shah** [reply]
instead of remaking everything, you can try migrating the data
we have the doc for it as well.

**Reply by ayush-shah** [reply]
instead of remaking everything, you can try migrating the data
we have the doc for it as well.

**Reply by ayush-shah** [reply]
https://docs.open-metadata.org/deployment/upgrade

**Reply by Ronaldo-spec** [reply]
@ayush-shah Thank you, Sir.

**Comment by ayush-shah** [comment]
I would recommend you to join our slack community as it would be way easier to interact, debug and have the issue fixed.
Here is the link: https://slack.open-metadata.org/
